scripts/components/FlCheck.ts

```diff
@@ -26,26 +27,20 @@ export default class FlCheck extends FlCheckDesign {
         this.__onToggleChange = value;
     }
     private toggleActiveInActive(value: boolean) {
+        setVisibility(this.imgCheck, value);
         if (value) {
             this.dispatch({
                 type: 'removeClassName',
                 className: '.flCheck.inactive'
             });
-            this.imgCheck.dispatch({
-                type: 'removeClassName',
-                className: '.flCheck-imgCheck.hidden'
-            });
             this.active = true;
         } else {
             this.dispatch({
                 type: 'pushClassNames',
                 classNames: '.flCheck.inactive'
             });
-            this.imgCheck.dispatch({
-                type: 'pushClassNames',
-                classNames: '.flCheck-imgCheck.hidden'
-            });
             this.active = false;
         }
+        this.applyLayout();
```

Comment:
Hmm I remember, we don't use applyLayout for android. Could you try use with only for iOS?

```ts
System.OS === System.OSType.IOS && this.applyLayout();
```

If everything will be okay, don't touch but if it will not work correctly, restore back.

Reply:
DONE.